### PR TITLE
Set the AES Key to configurable

### PR DESCRIPTION
**Pull request by w1543565388**

### CODE REVIEW

File: pkg/cli/client.go [hunk 1]
```diff
@@ -29,6 +29,12 @@ func NewClientCmd() *cli.Command {
 				Usage:    "password of virtuallan endpoint user",
 				Required: false,
 			},
+			&cli.StringFlag{
+				Name:     "key",
+				Aliases:  []string{"k"},
+				Usage:    "encryption key of virtuallan",
+				Required: false,
```
Comment (lucheng0127) [inline pkg/cli/client.go]: Required true

File: pkg/client/client.go [hunk 2]
```diff
@@ -68,6 +69,8 @@ func handleSignal(conn *net.UDPConn, sigChan chan os.Signal) {
 
 func Run(cCtx *cli.Context) error {
 	var user, passwd string
+	cipher.AESKEY = cCtx.String("key")
```
Comment (lucheng0127) [inline pkg/client/client.go:77]: If use wrong key, or invalidate key, weather the client gonna login timeout

File: pkg/server/server.go [hunk 3]
```diff
@@ -82,6 +85,10 @@ func Run(cCtx *cli.Context) error {
 
 	svc.ServerConfig = cfg
 	svc.userDb = filepath.Join(cfgDir, "users")
+	if ok := utils.ValidateKey(svc.Key); !ok {
```
Comment (lucheng0127) [inline pkg/server/server.go]: Add a new line will be better

File: pkg/utils/validate.go [hunk 4]
```diff
@@ -7,6 +7,14 @@ import (
 	"strings"
 )
 
+func ValidateKey(key string) bool {
```
Comment (lucheng0127) [inline pkg/utils/validate.go:10]: Why don't use it in aes.go NewAESCipher

**Review by lucheng0127 — CHANGES_REQUESTED** [review]
Key is required

File: pkg/config/config.go [hunk 5]
```diff
@@ -23,6 +24,7 @@ type ServerConfig struct {
 	IP        string `yaml:"ip"`
 	Bridge    string `yaml:"bridge"`
 	LogLevel  string `yaml:"log-level"`
+	Key       string `yaml:"key" validate:"required"`
```
Comment (lucheng0127) [inline pkg/config/config.go]: Use validate.RegisterValidation  to validate key
Also other filed is required except web

**Comment by w1543565388** [comment]
It has been modified according to the format

**Review by lucheng0127 — COMMENTED** [review]
LGTM

**Pull request merged**